swimlane/core/adapters/record.py

```diff
@@ -294,12 +321,14 @@ def bulk_modify(self, *filters_or_records, **kwargs):
         modifications = []
         for field_name, update_value in values.items():
             mod_field = record_stub.get_field(field_name)
+            if not mod_field.bulk_modify_support:
+                raise ValueError("Field '{}' of Type '{}', is not supported for bulk modify".format(field_name, mod_field.field_type[0]))
```

Comment:
Make type be the class name `mod_field.__class__.__name__` instead, will be more useful than the `$type` for the user.